relay/cmd/main.go

```diff
@@ -54,7 +54,7 @@ func RunRelay(ctx *cli.Context) error {
 	if err != nil {
 		return fmt.Errorf("failed to create logger: %w", err)
 	}
-	logger.Info(fmt.Sprintf("Relay configuration: %#v", config))
+	logger.Info(fmt.Sprintf("Relay configuration: %#v", config.SanitizedConfig()))
```

Comment:
let's make sure that none of the configs printed (including awsConfig, ethConfig, etc) don't leak any secrets, and if nothing is leaked maybe there's a comment we can add to shutup codeQL on this one if its a false positive

Reply:
I think this logging was meant for debugging. Maybe we can just log few specific fields instead of the whole struct?

Reply:
which fields are useful for debugging? we got 
```
RelayIDs []v2.RelayKey, 
GRPCPort int
MaxGRPCMessageSize int
MetadataCacheSize int
MetadataMaxConcurrency int
BlobCacheBytes uint64
BlobMaxConcurrency int
ChunkCacheSize uint64
ChunkMaxConcurrency int
MaxKeysPerGetChunksRequest int
RateLimits limiter.Config
AuthenticationKeyCacheSize int <- codeQL doesn't like this one
AuthenticationTimeout time.Duration
AuthenticationDisabled bool
Timeouts TimeoutConfig
OnchainStateRefreshInterval time.Duration
MetricsPort int
```